scripts/qa_crowbarsetup.sh

```diff
@@ -1267,6 +1267,14 @@ EOF
     # setup_base_images.rb is for SUSE Cloud 1.0 and update_nodes.rb is for 2.0
     sed -i -e 's/\(rootpw_hash.*\)""/\1"$2y$10$u5mQA7\/8YjHdutDPEMPtBeh\/w8Bq0wEGbxleUT4dO48dxgwyPD8D."/' /opt/dell/chef/cookbooks/provisioner/recipes/setup_base_images.rb /opt/dell/chef/cookbooks/provisioner/recipes/update_nodes.rb
 
+    if [[ $hacloud = 1 ]] ; then
+        f=/opt/dell/chef/cookbooks/nfs-server/templates/default/exports.erb
+        mkdir -p /var/lib/glance/images
+        if ! grep -q /var/lib/glance/images $f; then
+            echo "/var/lib/glance/images     <%= @admin_subnet %>/<%= @admin_netmask %>(rw,async,no_root_squash,no_subtree_check)" >> $f
```

Comment:
I agree, but maybe we need a separate story for that, since I guess it would more than 30 minutes (and backporting would be required if we wanted to use it for this use case, since we are testing HA on Cloud 5 and 6).